tests/integration/remote-test-firewall.sh

```diff
@@ -0,0 +1,411 @@
+#!/bin/bash -eEu
```

Comment:
This file is a system test. Could you put it under a `:tests/system/` directory. As is, I think it will be unintentionally executed by `make checkinstall` and by the `nilrt-snac-ptest` autotest - which doesn't seem to be what we want.

Reply:
Done.